### PR TITLE
Added abstract (undefined) tasks to process generator

### LINKED ISSUE #11319: Generate processes with undefined tasks
**Description**

Since version `8.2`, Zeebe supports BPMN undefined tasks (see https://github.com/camunda/zeebe/issues/10562).

We should extend our process generator to cover undefined tasks in our automated tests.

### PR DESCRIPTION
**Pull request by qfcgc**
## Description

In this PR, the process generator is extended, so now it generates undefined (abstract) tasks as well.

## Related issues

closes https://github.com/camunda/zeebe/issues/11319

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [x] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to [backport](https://github.com/camunda/zeebe/compare/stable/0.24...main?expand=1&template=backport_template.md&title=[Backport%200.24]) the fix to the last two minor versions. You can trigger a backport by assigning labels (e.g. `backport stable/1.3`) to the PR, in case that fails you need to create backports manually.

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [x] The behavior is tested manually
* [ ] The change has been verified by a QA run
* [ ] The impact of the changes is verified by a benchmark

Documentation:
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] If the PR changes how BPMN processes are validated (e.g. support new BPMN element) then the Camunda modeling team should be informed to adjust the BPMN linting.

Other teams:
If the change impacts another team an issue has been created for this team, explaining what they need to do to support this change.
- [ ] [Operate](https://github.com/camunda/operate/issues)
- [ ] [Tasklist](https://github.com/camunda/tasklist/issues)
- [ ] [Web Modeler](https://github.com/camunda/web-modeler/issues)
- [ ] [Desktop Modeler](https://github.com/camunda/camunda-modeler/issues)
- [ ] [Optimize](https://github.com/camunda/camunda-optimize/issues)

Please refer to our [review guidelines](https://github.com/camunda/zeebe/wiki/Pull-Requests-and-Code-Reviews#code-review-guidelines).

### CODE REVIEW

**Comment by berkaycanbc** [comment]
Hey @qfcgc, thanks for the contribution! I will take a look in my first availability. 👍

File: test-util/src/main/java/io/camunda/zeebe/test/util/bpmn/random/blocks/AbstractBlockBuilder.java [hunk 1]
```diff
@@ -26,6 +30,12 @@ protected AbstractBlockBuilder(final String prefix, final String elementId) {
     this.prefix = prefix;
   }
 
+  @Override
+  public AbstractFlowNodeBuilder<?, ?> buildFlowNodes(
+      final AbstractFlowNodeBuilder<?, ?> nodeBuilder) {
+    return nodeBuilder.manualTask(elementId).name(elementId);
```
Comment (berkaycanbc) [inline test-util/src/main/java/io/camunda/zeebe/test/util/bpmn/random/blocks/AbstractBlockBuilder.java:36]: ❌ Let's keep this in `ManualTaskBlockBuilder` class and create a new method in `AbstractFlowNodeBuilder`. Afterwards, we can call it here as `nodeBuilder.abstractTask(elementId).name(elementId);`. This way, the implementation will be aligned with other elements as well.

**Review by berkaycanbc — CHANGES_REQUESTED** [review]
Good job! 🚀 I asked for one change to make changes aligned with the current implementation of other components.

**Comment by abbasadel** [comment]
Hey @qfcgc, thanks again for the contribution. I just wanted to make sure that you got the chance to check the review from @berkaycanbc.

**Comment by berkaycanbc** [comment]
Since we did not get an update on this issue for a long time, I am closing this PR and moving the issue back to our backlog.

@qfcgc Please reopen the PR if you think you can further contribute on that based on the feedback.

Cc: @abbasadel

**Pull request closed without merging**